arho_feature_template/gui/components/code_combobox.py

```diff
@@ -90,3 +101,30 @@ def populate_from_code_layer(self, layer_type: type[AbstractCodeLayer]) -> None:
     def value(self) -> str:
         item = self.tree_widget.selectedItems()[0]
         return item.data(0, Qt.UserRole)
+
+    def set_value(self, value: str | None) -> None:
+        # NOTE: Does not work fully currently
```

Comment:
Tähän voisi lisätä kommentin, että miksi tää on tehty näin. Ja mikä ei tällä hetkellä toimi.